### PR TITLE
Add a bottom to the divider tray.

### PR DESCRIPTION
**Pull request by pokowaka**
This add support for the generation of a bottom for the divider tray.
The bottom can be generated by passing the --bottom=True flag.

This makes it possible to generate ./static/samples/DividerTray.jpg

### CODE REVIEW

File: boxes/generators/dividertray.py [hunk 1]
```diff
@@ -130,6 +137,11 @@ def render(self):
             max(facing_wall_length, side_wall_length), self.h, "ffff", move="right only"
         )
 
+        # Bottom piece.
+        if self.bottom:
+            self.rectangularWall(facing_wall_length, side_wall_length, "ffff",
+                    callback=[partial(self.generate_finger_holes, side_wall_length)], move="up")
```
Comment (florianfesti) [inline boxes/generators/dividertray.py]: This looks a bit fishy. Shouldn't the edge type be "e" for the sides that don't have a side wall attached? As happens when left_wall or right_wall is False?

Comment (pokowaka) [reply boxes/generators/dividertray.py]: It looks like you might be right :-).

**Comment by florianfesti** [comment]
Ok, fixed the side edges of the bottom and commited as a0338a559f55afa260b3beb1f7f0d3ea1caa55b5

**Comment by florianfesti** [comment]
Oh, thanks for the patch! There are several people eager to use this new feature:
https://hackaday.io/project/10649-boxespy/discussion-139426

**Pull request closed without merging**